src/OnboardingSPA/components/TextInput/TextInputSiteGen/index.js

```diff
@@ -54,6 +54,10 @@ const TextInputSiteGen = ( {
 		}
 	};
 
+	const getRemainingCharacterCount = () => {
+		return Math.max( 200 - ( customerInput?.length ?? 0 ), 0 );
```

Comment:
Is it a good idea to move the value `200` to flow data instead of hardcoding it here? This will allow us to update the count in a standard way in the future and also work well in case we decide to experiment with it.

Reply:
Can be taken up later.